src/calendar-list/item.js

```diff
@@ -47,13 +71,13 @@ class CalendarListItem extends Component {
           disabledByDefault={this.props.disabledByDefault}
           showWeekNumbers={this.props.showWeekNumbers}
           renderArrow={this.props.renderArrow}
-          onPressArrowLeft={this.props.onPressArrowLeft}
-          onPressArrowRight={this.props.onPressArrowRight}
+          onPressArrowLeft={this.onPressArrowLeft.bind(this)}
```

Comment:
Please don't use bind in props. You can use the arrow function on the methods' signatures. For example:
`onPressArrowRight = (_, month) => {
    ...
}`

Reply:
@Inbal-Tish I've made the requested changes. Please merge.